runtime/plugin/tohtml.lua

```diff
@@ -0,0 +1,473 @@
+--- Neovim plugin for converting a syntax highlighted file to HTML.
+
+if vim.g.loaded_2html_plugin then
+  return
+end
+vim.g.loaded_2html_plugin = 'nvim10.0_v1'
+
+---@class tohtml.state
+---@field bufnr number
+---@field background string
+---@field foreground string
+---@field title string|false
+---@field start_row number
+---@field end_row number
```

Comment:
Seconding @lucario387's comment, it should be clear whether this is zero-based and/or past-to-end convention. See #26631